src/main/java/org/schemaspy/progress/FinishCallback.java

```diff
@@ -0,0 +1,20 @@
+package org.schemaspy.progress;
+
+import java.time.Duration;
+
+/**
+ * Abstraction for reporting or acting
+ * upon the completion of a job, also
+ * it's provided with the number of
+ * progress calls and the duration of
+ * the job
```

Comment:
Unclear concern. I bet there's a simpler design, which does not require a callback.

Reply:
There is but it might require refactors in other places.

I believe that it's more of a task and a job is a series of task, so I hope to eventually reach a state were it's iterators of tasks.